src/GraphQL/Types/Composite/AutoRegisteringObjectGraphType.cs

```diff
@@ -75,5 +79,20 @@ protected virtual IEnumerable<PropertyInfo> GetRegisteredProperties()
             => AutoRegisteringHelper.ExcludeProperties(
                 typeof(TSourceType).GetProperties(BindingFlags.Public | BindingFlags.Instance).Where(x => x.CanRead),
                 _excludedProperties);
+
+        /// <summary>
+        /// Analyzes a property and returns an instance of <see cref="TypeInformation"/>
+        /// containing information necessary to select a graph type. Any <see cref="GraphQLAttribute"/>
```

Comment:
> Any \<see cref="GraphQLAttribute"/>

Not only those ,
https://github.com/graphql-dotnet/graphql-dotnet/pull/2884/files#r795050710

Reply:
Well, this method doesn't have anything to do with the description attribute, but it does read NRT attributes and the Required attribute.  I updated the comment.